apps/cookbook/src/app/showcase/accordion-showcase/accordion-showcase.component.ts

```diff
@@ -64,7 +66,7 @@ export class AccordionShowcaseComponent {
   ];
   events: ApiDescriptionEvent[] = [
     {
-      name: 'toggle',
+      name: 'toggled',
```

Comment:
Please revert so our docs correctly match the component input.
```suggestion
      name: 'toggle',
```